sklearn/cross_validation.py

```diff
@@ -632,11 +638,13 @@ class ShuffleSplit(BaseShuffleSplit):
     n_iter : int (default 10)
         Number of re-shuffling & splitting iterations.
 
-    test_size : float (default 0.1), int, or None
+    test_size : float (default 0.1), int, or None,
```

Comment:
Finally, I think you are right, it will make me puzzle if the docstring dosen't match the parameter list. It's enough to explain it afterwards.